drivers/fuel_gauge/sbs_gauge/sbs_gauge.c

```diff
@@ -100,10 +100,6 @@ static int sbs_gauge_get_prop(const struct device *dev, struct fuel_gauge_get_pr
 		rc = sbs_cmd_reg_read(dev, SBS_GAUGE_CMD_AVG_TIME2FULL, &val);
 		prop->value.runtime_to_full = val;
 		break;
-	case FUEL_GAUGE_SBS_MFR_ACCESS:
-		rc = sbs_cmd_reg_read(dev, SBS_GAUGE_CMD_MANUFACTURER_ACCESS, &val);
-		prop->value.sbs_mfr_access_word = val;
-		break;
```

Comment:
bit confused by the change, are you actually removing these and re-add them later? idea was just to define them in a separate header, no point ripping off the implementation

Reply:
I should've made this more explicit in the PR comment. I think working with the SBS properties directly are really the wrong level of abstraction here.

Originally, I was going to just move them to another header as we discussed, but looking at them again, I wasn't sure having SBS specific properties that were more references to the direct registers was a good idea.

It seems like @teburd thinks we definitely need SBS manufacturers access though. Which is fine, we could keep that or define it in an external `sbs.h` so it's not cluttering the existing properties.